files/nrpe/check_openstack.py

```diff
@@ -732,7 +732,7 @@ def check_floating_ips(self):
         start_fourth_octet = int(start_match.group(4))
         end_third_octet  = int(end_match.group(3))
         end_fourth_octet = int(end_match.group(4))
-        ips_total += end_fourth_octet - start_fourth_octet
+        ips_total += (end_third_octet - start_third_octet) * 256 + end_fourth_octet - start_fourth_octet
```

Comment:
1) I think there has been a longer standing bug with off-by-one error, so proposing to fix that in this same feature branch:

-ips_total += (end_third_octet - start_third_octet) * 256 + end_fourth_octet - start_fourth_octet
+ips_total += (end_third_octet - start_third_octet) * 256 + end_fourth_octet - start_fourth_octet + 1

2) Also maybe minor documentation enhancement to the function in general?

-# Count the number of ip's in each subnet
+# Count the number of ip's in each subnet's DHCP allocation pool